src/components/scenes/PluginViewScene.js

```diff
@@ -271,6 +344,8 @@ class PluginView extends React.Component<PluginProps, PluginState> {
                 this.openingSendConfirmation = false
               }
             }
+            console.log('a1: info', info)
```

Comment:
Extraneous debug logging. Please remove for production

Reply:
fixed